README.md

```diff
@@ -50,152 +48,76 @@ are automatically reflected on preliminary.istio.io.
 - The content of istio.io is taken from the latest release-XXX branch. The specific branch that
 is used is determined by the istio.io [Netlify](https://netlify.com) project's configuration.
 
-- The content of archive.istio.io is taken from the older release-XXX branches. The set of branches that
-are included on archive.istio.io is determined by the `TOBUILD` variable in this
-[script](https://github.com/istio/istio.io/blob/master/scripts/build_archive_site.sh).
-
 ### Publishing content immediately
 
 Checking in updates to the master branch will automatically update preliminary.istio.io, and will only be reflected on
 istio.io the next time a release is created, which can be several weeks in the future. If you'd like some changes to be
 immediately reflected on istio.io, you need to check your changes both to the master branch and to the
-current release branch (named release-XXX such as release-1.4).
+current release branch (named `release-<MAJOR>.<MINOR>` such as `release-1.7`).
 
 This process can be taken care of automatically by our infrastructure. If you submit a PR
 to the master branch and annotate the PR with the `actions/merge-to-release-branch` label,
 then as soon as your PR is merged into master, it will be merged into the current release branch.
 
-### Creating a version
+### Creating a major/minor release
 
 Here are the steps necessary to create a new documentation version. Let's assume the current
-version of Istio is 1.3 and you wish to introduce 1.4 which has been under development.
+version of Istio is 1.6 and you wish to introduce 1.7 which has been under development.
 
 #### When Istio source code is branched
 
-The documentation repo pulls content from the Istio source repos for inclusion in the published site.
-When the source repos are branched in preparation for a release, a few changes are needed in the
-documentation repo to track this:
-
-1. Switch to the **master** branch of the istio/istio.io repo and make sure everything is up to date.
-
-1. Edit the file `Makefile.core.mk` and change the `SOURCE_BRANCH_NAME` variable to the
-name of the newly created source branches (in this case `release-1.4`).
-
-1. Edit the file `data/args.yml` and set the `source_branch_name` field to the name of the newly created source
-branches (in this case `release-1.4`).
-
-1. Run `make update_all` in order to retrieve the latest material from the source repositories.
-
-1. Commit the previous edits to your local git repo and push your **master** branch to GitHub.
+Run `make prepare-1.7.0`, and that's it. This will grab the latest material from the new istio source branch.
```

Comment:
This sounds pretty mysterious. You should say what "latest material" it is grabbing and where's it's putting it. Isn't it it just updating the istio/istio.io master branch to use the release branch of istio/istio for doc testing? Or is there more? Either way, you should say exactly what "preparing" entails.